AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientObserver.swift

```diff
@@ -37,6 +39,12 @@ class DefaultAudioClientObserver: NSObject, AudioClientDelegate {
         let newAudioState = Converters.AudioClientState.toSessionStateControllerAction(state: audioClientState)
         let newAudioStatus = Converters.AudioClientStatus.toMeetingSessionStatusCode(status: status)
 
+        if newAudioState == .unknown {
```

Comment:
Actually, should this be `newAudioStatus == .unknown`?

Reply:
You are right, will send out a new PR